### DISCUSSION TITLE
Problem : mxbai-embed-large don't work !

### DISCUSSION THREAD

**Discussion by zouzeTG** [open-webui/open-webui#3502] [Q&A]
Hi I have set mxbai-embed-large model for RAG (Documents) and it don't work :-( . not any error, but chatbot say that it don't have any doc to analyse.

**Comment by justinh-rahb** [comment]
Make sure you also reset the vector storage when changing RAG models:

![Screenshot 2024-06-28 at 12 08 28 PM](https://github.com/open-webui/open-webui/assets/52832301/ea41a7b0-327f-455f-b855-aae7d5d367a5)
